polaris-tokens/src/token-groups/shadow.ts

```diff
@@ -82,4 +84,8 @@ export const shadow: {
     value:
       '0px 4px 6px -2px rgba(0, 0, 0, 0.2), inset 0px -1px 0px #D4D4D4, inset -1px 0px 0px #E3E3E3, inset 1px 0px 0px #E3E3E3, inset 0px 1px 0px #E3E3E3',
   },
+  'shadow-button-experimental': {
```

Comment:
New token for the box-shadow of the CTA in the `FileUpload` component. 
![Uploading Screenshot 2023-06-07 at 11.14.50 am.png…]()